khelo/lib/ui/flow/score_board/score_board_view_model.dart

```diff
@@ -796,7 +798,9 @@ class ScoreBoardViewNotifier extends StateNotifier<ScoreBoardViewState> {
       default:
         // fair_deliveries / 6 == number_of_over_decided
         // number_of_total_ball == total_played_ball
-        final totalBalls = (state.match?.number_of_over ?? 0) * 6;
+        final revisedOver = state.match?.revised_target?.overs;
+        final totalBalls =
+            (revisedOver ?? state.match?.number_of_over ?? 0) * 6;
```

Comment:
I don't think there are always 6 balls in an over because we haven't set a fixed number of balls per over.